lib/src/Base/Stat/CorrelationAnalysis.cxx

```diff
@@ -157,7 +157,7 @@ Point CorrelationAnalysis::computePCC(const Sample & sampleX,
                                       const Sample & sampleY) const
 {
   const UnsignedInteger dimension = sampleX.getDimension();
-  if (!(dimension >= 2)) throw InvalidDimensionException(HERE) << "Error: input sample must have dimension > 1, here dimension=" << dimension;
+  //if (!(dimension >= 2)) throw InvalidDimensionException(HERE) << "Error: input sample must have dimension > 1, here dimension=" << dimension;
```

Comment:
The PCC may not be very interesting in the case where `sampleX` is 1-dimensional because it is simply equal to the SRC, but the computation is correct.